WordPress/Docs/PHP/RestrictedPHPFunctionsStandard.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Restricted PHP Functions"
+    >
+    <standard>
+    <![CDATA[
+    The restricted function create_function() should not be used.
+
+    create_function() is deprecated as of PHP 7.2 and removed in PHP 8.0. Please use declared named or anonymous functions instead.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Using anonymous function (closure).">
+        <![CDATA[
+add_action( 'init', function() {
```

Comment:
I'm unsure if including a call to `add_action()` here helps understand what the sniff does or if it just adds noise. Usually, the examples are minimalistic, and just calling calling `create_function()` or calling it and assigning the returned value to a variable is enough?